tests/integration_tests/test_applications.py

```diff
@@ -13,12 +13,20 @@
 
```

Comment:
What is the motivation to re-arrange the sequence of the tests? It makes it hard to compare! 

If there is no strong reason for this, could we go back to the original sequence? The possibility to track changes is important.

Reply:
I sorted it alphabetically, as it is displayed in my IDE. it makes more sense to me than chronologically. since we work on the tests throughout time, the chronology loses its meaning. There is the change to organize it according to the module, but the names of the applications do not always refer directly to a module, hence, it would be also difficult. My final suggestion is to keep it alphabetically.

Reply:
You are right, the current sorting was kind of random. It just now almost impossible to compare the changes. I will have another look, but it makes reviewing much harder.

Reply:
Wait, the order was not random!
I ordered it such that the first 5-10 applications were roughly the quickest ones. Originally we had the optics first which were the slowest tests first and I always had to wait 5-10 minutes just to pass the first ones. The tests were also bunched in Camera/Optics/DB/etc. That I actually liked a lot. Could we go back to the order before this change?

Reply:
Ok, I just returned to the previous order